doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -455,3 +456,10 @@ which attributes to include in the response to the query) and unary attribute fi
 `expression` (to return only entities which have these attributes). Use them instead.
 
 [Top](#top)
+
+## Additional fields in `httpInfo` for custom notifications
+
+Orion supports a boolean field `includePayload` in `httpInfo` for custom notifications that can be used to avoid payload in the outgoing notification. If `includePayload` is set to `true` then no payload is included in the notification (and `payload` field, if used, is ignored).
+Default value if `includePayload` if omitted is `true`.
```

Comment:
```suggestion
Orion supports a boolean field `includePayload` in `httpInfo` for custom notifications that can be used to avoid payload in the outgoing notification. If `includePayload` is set to `false` then no payload is included in the notification (and `payload` field, if used, is ignored).
Default value if `includePayload` if omitted is `true` (that is, payload is included by default).
```

Note the default `true` will ensuse backward compatibility.

Reply:
Updated